JavaScript/Embeddings/SQL (for JS template).sublime-syntax

```diff
@@ -0,0 +1,14 @@
+%YAML 1.2
+---
+# http://www.sublimetext.com/docs/syntax.html
+# highlight tagged template strings
+scope: source.sql.js-template
+version: 2
```

Comment:
Those syntaxes don't need to be `version: 2` to be embeddable in JS/TS.

```suggestion
```

Reply:
> Well, it is a prerequisite for this,

No

Reply:
Oh, right. I assumed new syntaxes should be the current latest version but I can make it `version: 1` if it is preferable.